yt/yt/server/lib/rpc_proxy/api_service.cpp

```diff
@@ -328,7 +328,13 @@ IRowStreamDecoderPtr CreateRowStreamDecoder(
 {
     switch (rowsetFormat) {
         case NApi::NRpcProxy::NProto::RF_YT_WIRE:
-            return CreateWireRowStreamDecoder(std::move(nameTable));
+        {
```

Comment:
Let's add some helper for filling options with max near the definition of `TWireProtocolOptions`. Like `CreateUnlimitedWireProtocolOptions` or a class static method.